example/gin/main.go

```diff
@@ -0,0 +1,52 @@
+// (c) Copyright IBM Corp. 2021
+// (c) Copyright Instana Inc. 2016
+
+// +build go1.11
+
+package main
+
+import (
+	"flag"
+
+	"github.com/gin-gonic/gin"
+	instana "github.com/instana/go-sensor"
+	"github.com/instana/go-sensor/instrumentation/instagin"
+)
+
+const (
+	defaultPort    = "8881"
+	defaultAddress = "localhost"
+)
+
+func main() {
+	port := flag.String("port", defaultPort, "port to use by an example")
```

Comment:
A minor note: other examples have a slightly different way to specify the listen addr. I'd suggest we follow the same pattern: https://github.com/instana/go-sensor/blob/master/example/http-database-greeter/main.go#L35

Reply:
I will change and create a reminder for grpc-client-server example.